ui-v2/src/routes/automations/index.ts

```diff
@@ -1,6 +1,6 @@
 import { createFileRoute } from "@tanstack/react-router";
 
-export const Route = createFileRoute("/automations")({
+export const Route = createFileRoute("/automations/")({
```

Comment:
I'm pretty sure we don't want the trailing slash to match the other routes, but is there a reason the trailing slash is needed?
```suggestion
export const Route = createFileRoute("/automations")({
```

Reply:
I think its autogenerated from `tanstack router`, when you move it to a nested folder.
Not sure if thats intentional from the library

https://github.com/user-attachments/assets/76853980-7dd9-4832-be7d-523733048f9f

Reply:
https://github.com/user-attachments/assets/cbfb1da8-ea13-4d91-b5f0-d8f814ce1ef8